docs/gallery/plot_term_set.py

```diff
@@ -41,25 +41,61 @@
    For example, the NCBI Taxonomy is abbreviated as NCBI_TAXON, and Ensemble is simply Ensemble.
    As mentioned prior, the URI needs to be to the terms; this is to allow the URI to later be coupled
    with the source id for the term to create a valid link to the term source page.
-3. The schema uses LinkML enumerations to list all the possible terms. Currently, users will need to
-   manually outline the terms within the enumeration's permissible values.
+3. The schema uses LinkML enumerations to list all the possible terms. To define the all the permissible
+   values, the user can define them manually in the schema, transfer them from a Google spreadsheet,
+   or pull them into the schema dynamically from a LinkML supported source.
 
 For a clear example, please view the
 `example_term_set.yaml <https://github.com/hdmf-dev/hdmf/blob/dev/docs/gallery/example_term_set.yaml>`_
 for this tutorial, which provides a concise example of how a term set schema looks.
+
+.. note::
+    For more information on how to properly format the Google spreadsheet to be compatible with LinkMl, please
+    refer to https://linkml.io/schemasheets/#examples.
+
+.. note::
+    For more information how to properly format the schema to support LinkML Dynamic Enumerations, please
+    refer to https://linkml.io/linkml/schemas/enums.html#dynamic-enums.
 """
-######################################################
-# Creating an instance of the TermSet class
-# ----------------------------------------------------
 from hdmf.common import DynamicTable, VectorData
 import os
+import sys
+
+try:
+    import linkml_runtime  # noqa: F401
+except ImportError:
+    sys.exit(0)
+from hdmf.term_set import TermSet
 
 try:
     dir_path = os.path.dirname(os.path.abspath(__file__))
     yaml_file = os.path.join(dir_path, 'example_term_set.yaml')
+    schemasheets_folder = os.path.join(dir_path, 'schemasheets')
+    dynamic_schema_path = os.path.join(dir_path, 'example_dynamic_term_set.yaml')
 except NameError:
     dir_path = os.path.dirname(os.path.abspath('.'))
     yaml_file = os.path.join(dir_path, 'gallery/example_term_set.yaml')
+    schemasheets_folder = os.path.join(dir_path, 'gallery/schemasheets')
+    dynamic_schema_path = os.path.join(dir_path, 'gallery/example_dynamic_term_set.yaml')
+
+# Use Schemasheets to create TermSet schema
+# -----------------------------------------
+# The :py:class:`~hdmf.term_set.TermSet` class builds off of LinkML Schemasheets, allowing users to convert between
+# a Google spreadsheet to a complete LinkML schema. Once the user has defined the necessary LinkML metadata within the
```

Comment:
Can we add screenshot of an example Google spreadsheet here to illustrate how this looks? Or maybe even a small graphic that show the spreadsheet and the corresponding TermSet that LinkML creates?

Reply:
With how LinkML uses multiple tabs in the sheets, I think the link I added to look at schemasheets for users to have a look at an example sheets will be more helpful and give a better picture.